app/views/devise/sessions/_login_form.html.erb

```diff
@@ -1,6 +1,6 @@
-<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+<%= form_for(resource, as: resource_name, url: session_path(resource_name), validate:true) do |f| %>
 
-  <% if flash[:success] %>
+    <% if flash[:success] %>
```

Comment:
Lol why the added tab, I assume it's an automated styling that happened. Not a big deal.